spec/bolt_server/file_cache_spec.rb

```diff
@@ -145,7 +145,7 @@
     it 'will create and run the purge timer' do
       expect(file_cache.instance_variable_get(:@purge)).to be_a(Concurrent::TimerTask)
       expect(file_cache.instance_variable_get(:@cache_dir_mutex)).to eq(other_mutex)
-      expect(other_mutex).to receive(:with_write_lock).at_most(2).times
+      expect(other_mutex).to receive(:with_write_lock).at_most(3).times
```

Comment:
```suggestion
```
This is not a meaningful enough test to warrant yet another attempt to fix this race.  I think we can also safely delete the `sleep` on line 151 in addition to this assertion.